### DISCUSSION TITLE
Android support

### DISCUSSION THREAD

**Discussion by danboid** [Arkq/bluez-alsa#610] [General]
My situation is that I have a car stereo with an aux input but no bt support. I would like to be able to use my Android phone as a bt speaker when I have passengers with phones that have bt support but no headphone jack, thus enabling music played on their phone to be played on my car stereo via my phones jack output, all without installing any extra software on their phone.

I've had a pretty good search but I haven't been able to find an existing Android app that enables this, all I can find is audio streaming apps that use wifi for use with a PC.

Presuming that there isn't already an existing solution for this that has evaded my searches, might it be possible to add Android support to bluez-alsa? This would also really require writing some kind of Android GUI to configure bluez-alsa which may be outside the scope of this project but it may be possible for bluez-alsa to provide the backend for such an app with a few additions or modifications?

**Comment by arkq** [comment]
> might it be possible to add Android support to bluez-alsa?

If I remember correctly, someone was trying to use bluez-alsa with Android, but it was not vanilla Android. BlueALSA service is hard dependent on BlueZ >= 5, where vanilla Android uses BlueDroid (aka Fluoride)... That's the technical obstacle for running BlueALSA on Android, and I'm afraid that it's not easy to overcome it. The easiest way is to run Linux on Android device :D but I understand that that's not an option for you.
